src/main/java/com/epam/jwd/task01/logic/TariffLogic.java

```diff
@@ -0,0 +1,164 @@
+package com.epam.jwd.task01.logic;
```

Comment:
Looks like a candidate for adding to "service" package

Reply:
Agree